src/parse/mark.js

```diff
@@ -23,6 +36,17 @@ function parseMark(model, mark) {
   return mark;
 }
 
+// set color given in graph if "default" specified in spec for marks color
+function defaultColor(property, configProperty, prop, config) {
+  if (property in prop && 'value' in prop[property] && prop[property]['value'] === 'default') {
+    if (typeof config !== 'undefined' && 'marks' in config && configProperty in config.marks) {
+      prop[property]['value'] = config.marks[configProperty];
+    } else {
+      prop[property]['value'] = '#000000';
+    }
+  }
+}
+
```

Comment:
@arvind 
1. I named it defaultFill since I wanted to distinguish it from the regular fill that could be specified in the json spec.  I don't feel super strongly about having it be named defaultFill and not fill, it was more of a small decision I made along the way.  So if fill seems better to you, I can go ahead and change it!
2. I actually like that idea much more.  When I thought of default fill, my original thought was to have people specify 'default' which was why I went with that.  But agree that it makes more sense for users to be able to draw marks without specifying a color, in which case the default fill would be used instead.  Will work on changing that.